de-lib/src/main/java/org/iplantc/de/admin/desktop/client/metadata/view/OntologyLookupServiceParamListEditor.ui.xml

```diff
@@ -0,0 +1,56 @@
+<ui:UiBinder xmlns:ui='urn:ui:com.google.gwt.uibinder'
+             xmlns:c="urn:import:com.sencha.gxt.widget.core.client.button"
+             xmlns:container="urn:import:com.sencha.gxt.widget.core.client.container"
+             xmlns:form="urn:import:com.sencha.gxt.widget.core.client.form"
+             xmlns:grid="urn:import:com.sencha.gxt.widget.core.client.grid"
+             xmlns:toolbar="urn:import:com.sencha.gxt.widget.core.client.toolbar">
+
+    <ui:with field="appearance"
+             type="org.iplantc.de.admin.desktop.client.metadata.view.EditMetadataTemplateView.EditMetadataTemplateViewAppearance" />
+    <ui:with field="listStore"
+             type="com.sencha.gxt.data.shared.ListStore"/>
+    <ui:with field="cm"
+             type="com.sencha.gxt.widget.core.client.grid.ColumnModel"/>
+    <ui:with field="gridView"
+             type="com.sencha.gxt.widget.core.client.grid.GridView">
+        <ui:attributes forceFit="true"
+                       autoFill="true"/>
+    </ui:with>
+
+    <ui:with field="margins"
+             type="com.sencha.gxt.core.client.util.Margins">
+        <ui:attributes top="0" right="20" bottom="0" left="0"/>
+    </ui:with>
+    <ui:with field="layoutData"
+             type="com.sencha.gxt.widget.core.client.container.VerticalLayoutContainer.VerticalLayoutData">
+        <ui:attributes width="1" height="-1" margins="{margins}"/>
+    </ui:with>
+
+    <form:FieldSet ui:field="header" layoutData="{layoutData}" >
+        <container:VerticalLayoutContainer>
+            <container:child layoutData="{layoutData}">
+                <form:FieldLabel ui:field="helpLabel"
+                                 labelAlign="TOP"/>
+            </container:child>
+            <container:child layoutData="{layoutData}">
+                <toolbar:ToolBar>
+                    <c:TextButton ui:field="addBtn"
+                                  debugId='idAddBtn'
```

Comment:
I think we're leaning towards not using this type of debug ID because it's easy to create duplicate IDs this way.  I'm not sure how much testing goes into Belphy, but if we do want debug IDs, it should be through the `ensureDebugId()` methods that append names to the parent element names.

Reply:
Good catch. I think this was a case of clipboard inheritance from [SubmitAppForPublicUseView.ui.xml](https://github.com/cyverse-de/ui/blob/57d07ab9fdf546a59dfbe910c8477960e74c5216/de-lib/src/main/java/org/iplantc/de/apps/client/views/submit/SubmitAppForPublicUseView.ui.xml#L145-L153).

I look into updating this before merging 👍